util/random.go

```diff
@@ -0,0 +1,36 @@
+package util
+
+import (
+	"time"
+	"math/rand"
+)
+
+// Get a random time duration between the lower bound and upper bound. This is useful because some of our automated tests
+// wound up flooding the AWS API all at once, leading to a "Subscriber limit exceeded" error.
+func GetRandomTime(lowerBound, upperBound time.Duration) time.Duration {
+	if lowerBound == upperBound {
+		return lowerBound
+	}
```

Comment:
Ah, that's a good point, but what to return? I'll play around with a couple of options.